src/panels/lovelace/common/generate-lovelace-config.ts

```diff
@@ -112,9 +112,9 @@ export const computeSection = (
       ({
         type: "tile",
         entity,
-        show_entity_picture: ["person", "camera", "image"].includes(
-          computeDomain(entity)
-        ),
+        ...(["person", "camera", "image"].includes(computeDomain(entity))
+          ? { show_entity_picture: true }
+          : {}),
```

Comment:
Don't really care but might be a little easier to read? 🤷‍♂️ 
```suggestion
        show_entity_picture: ["person", "camera", "image"].includes(
          computeDomain(entity)
        ) || undefined,
```